server/controllers/cookieController.ts

```diff
@@ -0,0 +1,8 @@
+const cookieController = {
```

Comment:
I could definitely be wrong, but my understanding with the refresh tokens with google OAuth means that we don't have to do our own session management with cookies.